src/pricing/index.njk

```diff
@@ -375,6 +378,40 @@ hubspot:
     </div>
 </div>
 <!-- Form section -->
+<!-- Contact us -->
+<div id="contact-us" class="contentCloud hide w-full bg-gray-50 pt-0">
+    <div class="w-full bg-gray-50 pb-12">
+        <div class="container max-w-5xl m-auto grid grid-cols-1 md:grid-cols-2 px-6 pt-16 gap-x-16 gap-y-6">
+            <div>
+                <h3 class="w-full text-center md:text-left">
+                    Contact Us
+                </h3>
+                <p>
+                    To learn more about our plans, discuss your specific needs, or get advice on the best fit for your project, please fill out the form and we will get back to you as soon as possible.
+                </p>
+            </div>
+            {% if hubspot.cloud %}
```

Comment:
Why an if statement for a variable that's always set?

Reply:
You're right, I've removed the if statements